var/urepo/cgi/urepo-functions

```diff
@@ -24,7 +24,7 @@ generate_pkg_control() {
             return 1
             ;;
     esac
-    pkg_data=$(ar p "$pkg_path" "$control_file"|tar $tar_options - ./control) || {
+    pkg_data=$(ar p "$pkg_path" "$control_file"|tar --no-anchored $tar_options - control) || {
```

Comment:
I used ./control on purpose, otherwise file failed to extract. Not sure if something changed since then but please double check that this works fine using both packages from the upstream and generated by fpm.

Reply:
got it, thanks a bunch for confirmation Klaus!